koku/masu/api/urls.py

```diff
@@ -24,12 +26,14 @@
     update_charge,
     upload_normalized_data,
 )
-
+#ROUTER = DefaultRouter()
+#ROUTER.register(r'sources-status', SourceStatusViewSet, base_name='sources_status')
 urlpatterns = [
     url(r'^status/$', get_status, name='server-status'),
     url(r'^download/$', download_report, name='report_download'),
     url(r'^expired_data/$', expired_data, name='expired_data'),
     url(r'^report_data/$', report_data, name='report_data'),
     url(r'^update_charge/$', update_charge, name='update_charge'),
     url(r'^upload_normalized_data/$', upload_normalized_data, name='upload_normalized_data'),
+    url(r'^sources_status/$', SourceStatusView.as_view(), name='sources_status')
```

Comment:
I'm fine with this URL as long as we don't have other `sources/foo` style endpoints. If we have other sources endpoints, then I'd want to have this URL be `sources/status/$` or `status/sources/$` for consistency.